src/components/PluginScreenshot.jsx

```diff
@@ -37,16 +35,29 @@ function NewScreenshotPlugin() {
   };
 
   return (
-    <div>
-      <label>URL</label>
-      <input 
-        type="text"
-        placeholder="Enter URL"
-        id="urlInput"/>
-      <div>
-        <img id="screenshotImg" width="50%" />
+    <div className="flex flex-col items-center justify-center">
```

Comment:
Since you also changed the PluginScreenshot view, can you add a screenshot of how it looks like now?

Reply:
I created a commit with the new changes and will add the image to show how it looked like.